app/models/__init__.py

```diff
@@ -6,3 +6,4 @@
 from user import *  # noqa
 from location import *  # noqa
 from resource import *  # noqa
+from attribute import *  # noqa
```

Comment:
the `#noqa` is actually leftover from when we copied and pasted this from something else (I don't think there's a pep8 issue). We should stop putting it there.